Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1865,8 +1866,21 @@ def headerTransform(s): return stringEscapeMD(s, True, True)  # noqa
         sep = '---'
         mdResult += '|' + '|'.join([sep] * len(headers)) + '|\n'
         for entry in t:
-            vals = [stringEscapeMD((formatCell(entry.get(h, ''), False) if entry.get(h) is not None else ''),
+
+            entry_copy = entry.copy()
+
+            if date_fields:
+                for field in date_fields:
+
+                    try:
+                        entry_copy[field] = datetime.fromtimestamp(int(entry_copy[field])/1000).strftime('%Y-%m-%d %H:%M:%S')
+
+                    except BaseException:
```

Comment:
```suggestion
                    except Exception:
```
BaseException is also ok but a bit too broad, no reason to use it